packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -556,7 +563,15 @@ export class GarbageCollector implements IGarbageCollector {
              * but make it one day to be safe.
              */
             if (snapshotCacheExpiryMs !== undefined) {
-                this.sweepTimeoutMs = this.sessionExpiryTimeoutMs + snapshotCacheExpiryMs + oneDayMs;
+                const overrideSweepTimeoutMs =
```

Comment:
Since we're already able to override the sessionExpiry, snapshotCache, why don't we just override the oneDayMs wait? Maybe call it a buffer or something. It seems weird that that's the one value we don't override for our tests.

Reply:
Hmm interesting point.  So you're thinking to get a short sweepTimeout you'd set `TestOverride.SessionExpiryTimeoutMs` and `TestOverride.SweepTimeoutBufferMs` and `TestOverride.DisableSnapshoCache`?

As I'm thinking about it, the trouble is, sweep timeout is calculated using `this.sessionExpiryTimeoutMs` not the test override.  but maybe we can change that.

Another problem this brings to mind - As-is the test override will be nearly unusable; when creating a new file you'll have to update ContainerRuntimeOptions to pass in a correspondingly small value for sessionExpiry (this is what the tests do... I didn't think about how unrealistic this is in practice...).

I think this problem can be solved by updating the sessionExpiry TestOverride to be used for new files if set.  Then for new files, the TestOverride sessionExpiry applies and a shorter Sweep Timeout can be achieved - and I definitely agree it makes more sense to do that by also setting a short buffer, rather than setting a short Sweep Timeout and enforcing it to be > sessionExpiry.

Reply:
I'll make this change a bit later today.  Thanks for the feedback!

Reply:
We will be creating test files with the TestOverride flags, so if some person accesses these test files, it should be fine regardless. We want consistent behavior across clients regardless, it may be useful to test in a state with clients in different versions to imitate different clients.